src/js/export-favicon.js.rst

```diff
@@ -57,16 +59,40 @@ Start by ensuring a single layer has been selected as the favicon export source.
         sketch.alert('Select a layer to export as favicon.', 'Error');
       } else if (layers.length > 1) {
         sketch.alert('Select a single layer to export as favicon.', 'Error');
-      }
+      } else {
+
+Prompt the user for the output favicon file location.
+
+.. code-block:: javascript
+
+        const dialog = NSSavePanel.savePanel();
+        dialog.setTitle('Export Selected Layer as Favicon');
+        dialog.setNameFieldStringValue('favicon.ico');
+        dialog.setAllowedFileTypes([ 'ico' ]);
+        dialog.setShowsTagField(false);
+        dialog.setIsExtensionHidden(false);
+        dialog.setCanSelectHiddenExtension(true);
+
+        if (dialog.runModal() == NSOKButton) {
```

Comment:
`===`?

Reply:
Yup, this works. I'd presumed `===` wouldn't do since we've got Objective C objects here. https://github.com/turbobabr/Sketch-Plugins-Cookbook#cocoascript-dont-use--operator